### ISSUE TITLE
may i ask how to use this?

### ISSUE THREAD

**Issue by ArsenicX** [x0uid/SpotifyAdBlock#14]
The "Whitelist" link, i understand what that is. But the "blacklist", should i add them all into my hosts file? 3773 lines ?
I'm so confused.

**Comment by x0uid** [comment]
No, that's for [Pi-Hole](https://en.wikipedia.org/wiki/Pi-hole) users. 

for the regular OS hosts file u need [**this file**](https://raw.githubusercontent.com/x0uid/SpotifyAdBlock/master/hosts), just add all the text in the hosts file.

**Comment by ArsenicX** [comment]
wow thanks, it works.
But the bottom blank banner is a little annoying. I tried using Fiddler to catch the ad banner host address, and I did get two of them:
`wg.spotify.com`
`sdk1xyajs-data.xycdn.com`

I added this two host address into hosts file, and the blank ad banner disappeared about 15 mins.Then it appeared again. Apparently I failed. XD

Anyway, thanks for your collecting for the host address.
